### ISSUE TITLE
[Ubuntu] Leaving Emacs moves point out of mini-frame

### ISSUE THREAD

**Issue by zck** [muffinmad/emacs-mini-frame#25]
From `emacs -Q`, evaluate the following code:

```
(require 'package)
(package-initialize)
(require 'mini-frame)
(mini-frame-mode t)
```

Then:

1. Type `C-x b`. A mini-frame opens prompting to switch to a new buffer. Point is in that frame.
2. Change focus away from Emacs, to any other window. Focus could be changed with Alt-tab or the mouse.
3. Change focus back to Emacs. Again, this could be done with Alt-tab or the mouse.
4. Note that focus is no longer in the mini-frame, but back in the original window point was in _before_ we typed `C-x b`.

I'm running mini-frame from version 31380e7688c5e82965f72e5707cbff3ec2712ba9.

**Comment by muffinmad** [comment]
Yes. See [Bug#44415](https://debbugs.gnu.org/cgi/bugreport.cgi?bug=44415).
